pykokkos/core/cpp_setup.py

```diff
@@ -243,11 +243,17 @@ def invoke_script(self, output_dir: Path, space: ExecutionSpace, enable_uvm: boo
                               compute_capability,   # Device compute capability
                               lib_suffix,           # The libkokkos* suffix identifying the gpu
                               str(compiler_path)]   # The path to the compiler to use
-        compile_result = subprocess.run(command, cwd=output_dir, capture_output=True, check=False)
+
+        if sys.platform == "linux" or sys.platform == "linux2":
+            #on linux we can color the output otherwise this is not implemented
+            command = [string if string != '' else "''" for string in command]
+            command: Str = "script --log-io compile.out --return --command " + "\""+" ".join(command) + "\""
+
+        compile_result = subprocess.run(command, cwd=output_dir, capture_output=True, check=False, shell=True)
```

Comment:
For the `mod: float` example above, I see no improvement on this branch:

`Translation of <ast.FunctionDef object at 0x7f53c88238b0> failed`

Which failure exactly do you want me to try on `develop` vs. this branch?